pkg/state/state.go

```diff
@@ -168,7 +168,8 @@ BEGIN
 				),
 				'indexes', (
 				  SELECT json_object_agg(pi.indexrelid::regclass, json_build_object(
-				    'name', pi.indexrelid::regclass
+				    'name', pi.indexrelid::regclass,
+				    'unique', pi.indisunique
```

Comment:
are we missing the columns on what the index is created here?

Reply:
yes, that's coming in another PR.